ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/PermanentIdRequestViewImpl.java

```diff
@@ -154,7 +153,7 @@ public void onDialogHide(DialogHideEvent event) {
         ColumnConfig<PermanentIdRequest, String> nameCol = new ColumnConfig<>(pr_props.requestedBy(),
                                                                               appearance.nameColumnWidth(),
                                                                               appearance.nameColumnLabel());
-        ColumnConfig<PermanentIdRequest, String> pathCol = new ColumnConfig<>(pr_props.path(),
+        ColumnConfig<PermanentIdRequest, String> pathCol = new ColumnConfig<>(new PermanentIdRequestPathProvider(appearance),
```

Comment:
I think you can still inject`PermanentIdRequestPathProvider` in `PermanentIdRequestViewImpl`'s constructor like you have with the `appearance` and 'factory', but it's not critical.